README.md

```diff
@@ -1,30 +1,45 @@
-# Virtual list
+# &lt;virtual-list&gt;
 
-Implementation of a Virtual list API, influenced by the research done by the [infinite list study group](https://github.com/domenic/infinite-list-study-group).
+`<virtual-list>` renders only the visible portion of a list and optimizes the amount of DOM operations and reflows.
 
-## Local Setup
+## Properties
 
-Ensure you have installed the npm dependencies and serve from the project root
-```sh
-$ npm install
-$ polymer serve --npm
-```
-Then, navigate to the url: http://localhost:8081/components/virtual-list/
+- _newChild_
+  - type: `function(item: any, index: number) => Element`
+  - returns an `Element` for the given item and index.
+- _updateChild_

```

Comment:
Based on the descriptions, maybe better names for these would be childConnected and childDisconnected? I wonder if they should even be events. (But that can be a separate discussion after this PR is merged...)